lib/recurly/apple-pay.js

```diff
@@ -19,6 +21,28 @@ export function factory (options) {
 };
 
 /**
+ * Fields that are sent to API.
+ *
+ * @type {Array}
+ * @private
+ */
+
+var fields = [
+  'first_name',
+  'last_name',
+  'address1',
+  'address2',
+  'country',
+  'city',
+  'state',
+  'postal_code',
+  'phone',
+  'vat_number',
+  'fraud_session_id',
+  'token'
+];
```

Comment:
Agreed. I just didn't see a simple way to do it and wanted to get something out asap for Karen to test.